Sources/Fuzzilli/Minimization/BlockReducer.swift

```diff
@@ -82,6 +86,18 @@ struct BlockReducer: Reducer {
         
         verifier.tryNopping(candidates, in: program)
     }
+
+    private func reduceCodeString(codestring: Block, in program: Program, with verifier: ReductionVerifier){
+        var candidates = [Int]()
+        // Append the begin and end of the code string
+        candidates.append(codestring.head)
+        candidates.append(codestring.tail)
+
+        // Append the callFunction instruction which should immediately follow the EndCodeString instruction
```

Comment:
It doesn't always have to follow the EndCodeString instruction, e.g. if other code was inserted in between by some mutation (also, I think in the current case it's actually the `let eval = b.loadBuiltin("eval")` instruction, so you probably need `.tail + 2`?). However, it's probably not worth the effort to try and reduce all possible CodeString + eval call patterns. But maybe you can just test if the instruction you expect to be the `eval` call is actually a call instruction with the code string as first argument and otherwise bail?

Reply:
done, I've added the checks as requested and the reducer will now default to generic block reduction if it can't find the call to `eval()`.